core/shared/platform/include/platform_common.h

```diff
@@ -118,6 +125,16 @@ typedef void *(*thread_start_routine_t)(void *);
 #define bh_socket_t int
 #endif
 
+/* Get current resolution of clock id returns BHT_OK on success and BHT_ERROR on
+ * failure */
```

Comment:
```suggestion
/**
 * Get a resolution of the clock
 *
 * @param clock_id clock identifier
 * @param resolution output variable to store the clock resolution
 * @return BHT_OK if success; otherwise, BHT_ERROR
 */
```
please do the same for `os_clock_time_get`